front/src/components/messaging/MessageChannelItem.tsx

```diff
@@ -0,0 +1,77 @@
+import { FC } from 'react';
+import { Image, StyleSheet, View } from 'react-native';
+import { Text, TouchableRipple } from 'react-native-paper';
+
+import { MessageChannel } from '@/models/entities/messageChannel';
+
+/**
+ * The styles for the MessageChannelItem component.
+ */
+const styles = StyleSheet.create({
+  container: {
+    gap: 16,
+  },
+  horizontalContainer: {
+    flexDirection: 'row',
+    gap: 12,
+  },
+  nameText: {
+    marginBottom: 5,
+  },
+  profilePicture: {
+    borderRadius: 48,
+    height: 60,
+    width: 60,
+  },
+  verticalCenter: {
+    marginBottom: 'auto',
+    marginTop: 'auto',
+  },
```

Comment:
You can use flexbox properties instead:

```suggestion
  verticalCenter: {
    justifyContent: 'center',
  },
```